brvs-rules/src/main/java/iroha/validation/rules/impl/whitelist/WhitelistUtils.java

```diff
@@ -0,0 +1,146 @@
+package iroha.validation.rules.impl.whitelist;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.reflect.TypeToken;
+import iroha.protocol.QryResponses;
+import java.security.KeyPair;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import jp.co.soramitsu.iroha.java.IrohaAPI;
+import jp.co.soramitsu.iroha.java.Query;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class WhitelistUtils {
+
+  private static final Logger logger = LoggerFactory.getLogger(WhitelistUtils.class);
+
+
+  public static final String ETH_WHITELIST_KEY = "eth_whitelist";
+  public static final String BTC_WHITELIST_KEY = "btc_whitelist";
+
+  static final Map<String, String> assetToWhitelistKey;
```

Comment:
can we make it private?

Reply:
Nope, it is used to determine which asset belongs to what whitelist key (sora belongs to ether whitelist).